stdlib/REPL/src/latex_symbols.jl

```diff
@@ -211,27 +228,24 @@ const latex_symbols = Dict(
     "\\euro" => "€",
 
 # 732 symbols generated from unicode.xml
-    "\\textexclamdown" => "¡",
+    "\\exclamdown" => "¡",
     "\\sterling" => "£",
     "\\yen" => "¥",
-    "\\textbrokenbar" => "¦",
+    "\\brokenbar" => "¦",
     "\\S" => "§",
-    "\\textasciidieresis" => "¨",
     "\\copyright" => "©",
-    "\\textordfeminine" => "ª",
+    "\\ordfeminine" => "ª",
     "\\neg" => "¬",
     "\\circledR" => "®",
-    "\\textasciimacron" => "¯",
     "\\degree" => "°",
     "\\pm" => "±",
-    "\\textasciiacute" => "´",
```

Comment:
Also here --- `\acute` is a combining character but this one isn't.

Reply:
See above — not clear why we need tab-completion for the non-combining `´`